src/__tests__/index.js

```diff
@@ -196,16 +152,15 @@ import Demo from './demo'
 <Demo />
   `.trim()
 
-  const error = (await bundleMDX(mdxSource, {
+  const error = /** @type Error */ (await bundleMDX(mdxSource, {
     files: {
       './demo.tsx': `import './blah-blah'`,
     },
-  }).catch(e => e)) as Error
+  }).catch(e => e))
 
-  expect(error.message).toMatchInlineSnapshot(`
-    "Build failed with 1 error:
-    __mdx_bundler_fake_dir__/demo.tsx:1:7: error: [inMemory] Could not resolve \\"./blah-blah\\" in \\"./demo.tsx\\""
-  `)
+
+  assert.snapshot(error.message, `Build failed with 1 error:
+__mdx_bundler_fake_dir__/demo.tsx:1:7: error: [inMemory] Could not resolve "./blah-blah" from "./demo.tsx"`)
```

Comment:
This test is failing on Windows. Changing to:

```js
assert.snapshot(error.message, `Build failed with 1 error:
__mdx_bundler_fake_dir__/demo.tsx:1:7: error: [inMemory] Could not resolve "./blah-blah" from ".${path.sep}demo.tsx"`)
```

does fix the problem, however I suspect any automatic updates to the snapshot will break it again in the future.

Reply:
Fixed 👍

uvu doesn't auto-update snapshots, but I switched it to `.equal` anyway. Thanks!